api/src/test/java/org/apache/iceberg/types/TestReadabilityChecks.java

```diff
@@ -362,7 +365,7 @@ public void testStructWriteReordering() {
     List<String> errors = CheckCompatibility.writeCompatibilityErrors(read, write);
     Assert.assertEquals("Should produce 1 error message", 1, errors.size());
 
-    System.err.println(errors);
```

Comment:
Printing in tests should be removed rather than logged.

Reply:
removed